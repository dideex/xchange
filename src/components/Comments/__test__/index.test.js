// import React from 'react'
// import Comment from '../index'
// import {shallow, mount, render} from 'enzyme'

describe("Comments component", () => {
  it('jest', () => {
    expect(1 + 1).toBe(2)
  })
})