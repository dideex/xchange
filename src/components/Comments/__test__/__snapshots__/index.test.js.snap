// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Base markup 1`] = `"<section class=\\"css-uvb5el\\"><h2 class=\\"css-x184e\\"><span>Feedback from our customers</span></h2><div class=\\"css-rdbcp7\\"><div class=\\"slick-slider slick-initialized\\" dir=\\"ltr\\"><svg style=\\"display: block; fill: #191716;\\" class=\\"chevron__left\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill: #191716;\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg><div class=\\"slick-list\\"><div class=\\"slick-track\\" style=\\"opacity: 1;\\"><div data-index=\\"-2\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"./img/comment3.jpg\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Петухов В.М.</h3><p>Старший менеджер</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span> Но всегда возвращался сюда, и наверное, уже не буду пробывать другие.</span></p></div></article></div></div><div data-index=\\"-1\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"./img/comment4.jpg\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Петухов В.М.</h3><p>Старший менеджер</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span> Но всегда возвращался сюда, и наверное, уже не буду пробывать другие.</span></p></div></article></div></div><div data-index=\\"0\\" class=\\"slick-slide slick-active slick-current\\" tabindex=\\"-1\\" aria-hidden=\\"false\\" style=\\"outline: none; width: 0px;\\"><div><article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"./img/comment1.jpg\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Петухов В.М.</h3><p>Старший менеджер</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span>Работаю с этой компанией уже пять лет, много раз выручала. Перепробывал множество других обменников, но всегда возвращался сюда, и наверное, уже не буду пробывать другие.</span></p></div></article></div></div><div data-index=\\"1\\" class=\\"slick-slide slick-active\\" tabindex=\\"-1\\" aria-hidden=\\"false\\" style=\\"outline: none; width: 0px;\\"><div><article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"./img/comment2.jpg\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Петухов В.М.</h3><p>Старший менеджер</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span> Перепробывал множество других обменников, но всегда возвращался сюда, и наверное, уже не буду пробывать другие.</span></p></div></article></div></div><div data-index=\\"2\\" class=\\"slick-slide\\" tabindex=\\"-1\\" aria-hidden=\\"true\\" style=\\"outline: none; width: 0px;\\"><div><article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"./img/comment3.jpg\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Петухов В.М.</h3><p>Старший менеджер</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span> Но всегда возвращался сюда, и наверное, уже не буду пробывать другие.</span></p></div></article></div></div><div data-index=\\"3\\" class=\\"slick-slide\\" tabindex=\\"-1\\" aria-hidden=\\"true\\" style=\\"outline: none; width: 0px;\\"><div><article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"./img/comment4.jpg\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Петухов В.М.</h3><p>Старший менеджер</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span> Но всегда возвращался сюда, и наверное, уже не буду пробывать другие.</span></p></div></article></div></div><div data-index=\\"4\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"./img/comment1.jpg\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Петухов В.М.</h3><p>Старший менеджер</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span>Работаю с этой компанией уже пять лет, много раз выручала. Перепробывал множество других обменников, но всегда возвращался сюда, и наверное, уже не буду пробывать другие.</span></p></div></article></div></div><div data-index=\\"5\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"./img/comment2.jpg\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Петухов В.М.</h3><p>Старший менеджер</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span> Перепробывал множество других обменников, но всегда возвращался сюда, и наверное, уже не буду пробывать другие.</span></p></div></article></div></div><div data-index=\\"6\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"./img/comment3.jpg\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Петухов В.М.</h3><p>Старший менеджер</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span> Но всегда возвращался сюда, и наверное, уже не буду пробывать другие.</span></p></div></article></div></div><div data-index=\\"7\\" tabindex=\\"-1\\" class=\\"slick-slide slick-cloned\\" aria-hidden=\\"true\\" style=\\"width: 0px;\\"><div><article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"./img/comment4.jpg\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Петухов В.М.</h3><p>Старший менеджер</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span> Но всегда возвращался сюда, и наверное, уже не буду пробывать другие.</span></p></div></article></div></div></div></div><svg style=\\"display: block; fill: #191716;\\" class=\\"chevron__right\\" x=\\"0px\\" y=\\"0px\\" viewBox=\\"0 0 400 400\\"><path style=\\"fill: #191716;\\" d=\\"M184.9,344.1L8.2,167.4c-8.5-8.5-8.5-22.3,0-30.9l20.6-20.6c8.5-8.5,22.3-8.5,30.8,0l140.7,140.1l140.7-140.1 c8.5-8.5,22.3-8.5,30.8,0l20.6,20.6c8.5,8.5,8.5,22.3,0,30.9L215.8,344.1C207.3,352.6,193.5,352.6,184.9,344.1L184.9,344.1z\\"></path></svg></div></div></section>"`;
